test-network-function/lifecycle/suite.go

```diff
@@ -86,10 +86,7 @@ var drainTimeout = time.Duration(drainTimeoutMinutes) * time.Minute
 //
 var _ = ginkgo.Describe(common.LifecycleTestKey, func() {
 	configData := common.ConfigurationData{}
-	ginkgo.BeforeSuite(func() {
-		common.Loadconfiguration(&configData)
-		log.Info(configData.ContainersUnderTest)
-	})
+	configData.SetNeedsRefresh()
```

Comment:
Why do we need to force it here?

Reply:
Oh, I recall the singleton discussion now. We should be able to remove this once the extra layer of needsRefresh is eliminated, right?

Reply:
Yes. Once all the config is collapsed, we'll work with singleton and expose a single API to update everything.